docs/examples/exporting_models_exclude4.py

```diff
@@ -0,0 +1,26 @@
+from pydantic import BaseModel, Field, SecretStr
+
+
+class User(BaseModel):
+    id: int
+    username: str  # overridden by explicit exclude
+    password: SecretStr = Field(exclude=...)
```

Comment:
I agree. It would probably make sense to have this all the way down the stack and in the other forms (explicit `.dict` exclude for example) as well.  Otherwise, I would find it a bit confusing if `True` were to work sometimes but not other times.

I guess we can't get rid of using `...` though because of backwards compatibility right? Should we just allow both?

```
model.dict(exclude={'test': ...}) == model.dict(exclude={'test': True}) 
model.dict(exclude={'test': {'subfield': True}}) == model.dict(exclude={'test': {'subfield': True}}) 
Field(exclude=True) == Field(exclude=...)
```
?

then discuss if we want to drop the `...` in v2?

Reply:
yes, makes sense.

Reply:
:+1:  thanks! Should be ready from my point of view then. 

A small Question for future PRs: Am I supposed to change the label "awaiting author revision" when the revisions are addressed or just resolve the discussions involved? I could add something to the "contributing" page if you have a preferred flow (and you think it would help cut down on pings if that sort of thing is a problem).